source/includes/cxi/_segments.md

```diff
@@ -0,0 +1,43 @@
+# Segments/Properties
+
+### HTTP Request
+
+`PUT https://cxi-api.wootric.com/v1/segments`
+
+## Response segments bulk update
+
+Parameter | Type | Description
+--------- | ---- | -------
+resposes[][id] | integer | Response ID
+response[][custom_properties] | Array | list of properties(Hash { key: property_name, value: new_value})
+response[][custom_properties][][key] | string | name of the segment/property
+response[][custom_properties][][value] | string | new value for the segment/property
+
+#### Update bulk
+
+This endpoint updates a list of segment associated to a response
+
+```shell
+curl -s -H "Authorization: Bearer ACCESSTOKEN" -X PUT "https://cxi-api.wootric.com/v1/segments/update_bulk" \
+  -F "resposes=[id: RESPONSE_ID, custom_properties: [{key: PROPERTY_NAME, value: NEW_VALUE},{key: PROPERTY_NAME, value: NEW_VALUE}]]"
```

Comment:
`responses` :)

Reply:
Same

Reply:
Same here, it says `-F "resposes=[id: RESPONSE_ID` it should be `-F "responses=[id: RESPONSE_ID`, right?